src/tpm2/Object.c

```diff
@@ -460,7 +460,10 @@ ObjectLoad(
     object->publicArea = *publicArea;
     // If there is a sensitive area, load it
     if(sensitive == NULL)
-	object->attributes.publicOnly = SET;
+	{
+	    object->attributes.publicOnly = SET;
+	    MemorySet(&object->sensitive, 0, sizeof(object->sensitive));
```

Comment:
This line is necessary due to modifications by libtpms that affect how the state of OBJECTs is marshalled. Can you add a comment here `// needed for libtpms`.

Thank you!